### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by ghost** [physics-hacker/physicshacker.com#9]
All of the source code should, per GPL and established standard, contain a license header such as that below: (Replacing <file name> and <file description> with relevant data.) While I understand that you desire to keep the source code simple for the ease of usage by the students, all source code should have a license header on it. One example where this is necessary is what happens if a student gives the code to their friend. If the original student fails to supply a copy of the GPL to their friend, they have just committed copyright infringement. (Which is a federal offense) This also creates a massive legal mess for the student who received the code should they decide to share it as they are also - unknowingly - committing copyright infringement. While this may seem abstract and a distant issue, I personally know several people who have had issues with their code being stolen due to a failure to properly indicate its licensing. 

(I know you guys are probably sick of me going on about this, so this will be the last time I bring it up. I just need a written record [ for my own protection ] that I have notified the group about issues I see in how the system is copyrighted. )

TL;DR, While not strictly required, it is bad form and extremely rude to refrain from putting a copyright/copyleft notice on all copies of the source code and can cause a host of legal problems. 

/*
  <file name>
  <file description>

  Copyright (C) 2015-2016 Christopher Orban

  This program is free software; you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation; either version 3 of the License, or
  (at your option) any later version.

  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License
 along with this program; if not, see http://www.gnu.org/licenses/.
 */

**Comment by chrisorban** [comment]
We can add it to the functions.pde files. For the files with the draw function, we can just mention the the code is licensed under GPLv3.
